code/modules/food_and_drinks/recipes/processor_recipes.dm

```diff
@@ -90,6 +90,11 @@
 	input = /obj/item/food/grown/potato/wedges
 	output = /obj/item/food/fries
 
+/datum/food_processor_process/tempeh
```

Comment:
This recipe won't work, soybeans already process into soy dope via the food processor (see /datum/food_processor_process/soybeans).

Reply:
> This recipe won't work, soybeans already process into soy dope via the food processor (see /datum/food_processor_process/soybeans).

yes, I'm planning on changing the recipe, so instead of just soybean, it needs a mix of soybean and mushroom seed (maybe by mixing them in a bowl), then food process that mix instead.

Reply:
I added the tempeh blend item for food processor instead of soybean